packages/@uppy/transloadit/src/index.ts

```diff
@@ -238,6 +222,46 @@ declare module '@uppy/utils/lib/UppyFile' {
   }
 }
 
+const sendErrorToConsole = (originalErr: Error) => (err: Error) => {
+  const error = new ErrorWithCause('Failed to send error to the client', {
+    cause: err,
+  })
+  // eslint-disable-next-line no-console
+  console.error(error, originalErr)
+}
```

Comment:
Why move this? That seems to artificially increase the diff / polute the git blame with no clear benefit

Reply:
I had to? I deleted `AssemblyOptions`, as you can see in the diff. We need this here now.

Reply:
Half of the diff is from functions moved from the deleted file `AssemblyOptions`. I moved to rest to have all functions in a single place instead of between types and variables. slightly increased diff, but worth it.